database/migration/aruha-522-timelines.sql

```diff
@@ -0,0 +1,23 @@
+SET ROLE zalando_nakadi_data_owner;
+
+CREATE TABLE zn_data.storage (
+  st_id            VARCHAR(36) NOT NULL PRIMARY KEY,
+  st_type          VARCHAR(32) NOT NULL,
+  st_configuration JSONB       NOT NULL
+);
+
+
```

Comment:
@bocytko we didn't work with legacy databases at Zalando so this rule is not clear for us. Since in our codebase all tables had 2 letters, I thought the rule was to have 2 letters. So I asked to change from `t_id` to `tl_id` 😓